### ISSUE TITLE
[Features] Request for `skiptotime` command

### ISSUE THREAD

**Issue by Pizzarous** [stegripe/rawon#1120]
### Is your feature request related to a problem? Please describe.

# Request for `skiptotime` Command
As a user of the music bot, I would like to request for the addition _if possible_ of a `skiptotime` command, which would allow to skip to a specific time in the current playing song.

The `skiptotime` command would accept a single argument, which is the time (in seconds) to which the song should be skipped. For example, `!skiptotime 30` would skip the song to 30 seconds.

### Describe the solution you'd like

The `skiptotime` command would accept a single argument, which is the time (in seconds) to which the song should be skipped. For example, `!skiptotime 30` would skip the song to 30 seconds.

- If the command is executed successfully, the bot should skip to the specified time in the song and continue playing the song from that point.
- This feature would greatly improve the user experience and provide more flexibility for music listening.

### Describe alternatives you've considered

An alternative if we can't change the song current time could be grabbing the time from the URL and starting the song directly from that time
- youtu.be/dQw4w9WgXcQ`?t=30`
- youtube.com/watch?v=dQw4w9WgXcQ`&t=30s` 

`!play https://www.youtube.com/watch?v=dQw4w9WgXcQ&t=30s` would start the song 30 seconds in to it

### Thanks for your consideration.

**Comment by Mednoob** [comment]
`seek` command (or as you called it `skiptotime` command) is actually my current concern for this project. I've thought about this feature since I started working on the filter feature back some weeks ago (since I don't like the idea that the bot have to play the song from the start when enabling/disabling filter).

This feature is actually **could** be implemented. However, this will probably only work with `yt-dlp` stream strategy. I can't find how to make it work with `play-dl`. `play-dl` does have `seek` option to play from certain point. But, when I tried to run it, the bot didn't play any song and just immediately say that the song is done playing.

So, yeah. This feature currently can't be implemented. But don't worry, I'll notify when it's possible.
